docs/website/layouts/index.html

```diff
@@ -705,7 +705,8 @@ <h2>Declarative Policy</h2>
 
 # Employees can see all pets.
 allowed_pets[pet] {
-    [header, payload, signature] := io.jwt.decode(input.token)
+    #[header, payload, signature] := io.jwt.decode(input.token)
+    [_, payload, _] := io.jwt.decode(input.token)
```

Comment:
Let's remove the commented-out lines, please.

Reply:
ok.
I left it there because it will help people to understand the meaning of return values of `io.jwt.decode()`

Reply:
now comment line are removed. thanks

Reply:
🤔 It's a little unfortunate that you cannot do
```rego
[_header, payload, _signature] := io.jwt.decode(input.token)
```
in strict mode.

Reply:
hmm.. new syntax(`_$variable`) learned . thanks .

Reply:
yeah it's valid but it currently won't affect the compiler's strict mode: my example snippet would also be rejected. I think allowing for unused vars if they start with an underscore would perhaps be a nice enhancement.